src/main/resources/application.properties

```
@@ -1 +1,9 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/rick-and-morty
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.username=root
+spring.datasource.password=Bs1234!
```

sensitive data and credentials should not be passed to the open sources
```suggestion
spring.datasource.username=
spring.datasource.password=
```